PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml

```diff
@@ -23,6 +23,7 @@
   <!-- uncomment to enable plugin in all products
   <depends>com.intellij.modules.lang</depends>
   -->
+  <depends>com.intellij.modules.platform</depends>
```

Comment:
Specifically we need that for Rider - new cross platform .Net IDE based on top of Intellij.
jetbrains.com/rider